http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/DevModeHttpsIT.java

```diff
@@ -0,0 +1,90 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import com.gargoylesoftware.htmlunit.SilentCssErrorHandler;
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.DomElement;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+import io.quarkus.test.bootstrap.DevModeQuarkusService;
+import io.quarkus.test.bootstrap.Protocol;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.DevModeQuarkusApplication;
+import io.quarkus.test.services.URILike;
+import io.quarkus.test.utils.AwaitilityUtils;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@QuarkusScenario
+public class DevModeHttpsIT {
+    private static final String PROPERTY = "quarkus.qe.test.value";
+
+    @DevModeQuarkusApplication(ssl = true)
+    static RestService app = new DevModeQuarkusService()
+            .withProperty("quarkus.oidc.enabled", "false")
+            .withProperty("quarkus.keycloak.policy-enforcer.enable", "false")
+            .withProperty("quarkus.keycloak.devservices.enabled", "false");
+
+    private WebClient webClient;
+
+    @BeforeEach
+    void setUp() {
+        webClient = new WebClient();
+        webClient.setCssErrorHandler(new SilentCssErrorHandler());
+        webClient.getOptions().setUseInsecureSSL(true);
+        webClient.getOptions().setRedirectEnabled(true);
+        webClient.getOptions().setWebSocketEnabled(false);
+    }
```

Comment:
Could you also add the following properties (just to be sure that the cache doesn't play any role)
```
 webClient.getCookieManager().clearCookies();
        webClient.getCache().clear();
        webClient.getCache().setMaxSize(0);
```


Also, it could make sense, these other properties: 
```
webClient.getOptions().setDownloadImages(false);
        webClient.getOptions().setGeolocationEnabled(false);
        webClient.getOptions().setAppletEnabled(false);
        webClient.getOptions().setCssEnabled(false);
        webClient.getOptions().setThrowExceptionOnScriptError(false);
        webClient.getOptions().setThrowExceptionOnFailingStatusCode(false);
        webClient.getOptions().setRedirectEnabled(true);
```

Feel free to custom the HttpClient with more fine grain configuration

Reply:
Good idea, added.  I decided against two `.setThrowException` options, since we would want to fail early if something is wrong. And redirect was already enabled (line 39)